src/Form/WebPageArchiveDeleteForm.php

```diff
@@ -37,7 +37,15 @@ public function getConfirmText() {
    */
   public function submitForm(array &$form, FormStateInterface $form_state) {
     $this->entity->delete();
-    drupal_set_message($this->t('Web page archive %label has been deleted.', array('%label' => $this->entity->label())));
+
+    drupal_set_message(
+      $this->t('content @type: deleted @label.',
+        [
+          '@type' => $this->entity->bundle(),
+          '@label' => $this->entity->label(),
+        ]
+        )
```

Comment:
This file was generated by drupal console. These messages will need to be modified later anyway, so you cool with just addressing down the road?